docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -60,7 +60,7 @@ Featuring lines, areas, bars, pie charts and scatter plots, the latest versions
   Customize the legend text to your heart's content, making your charts more readable and visually
   appealing.
 
-Get started with charts [now](/x/react-charts/#getting-started)!
+Get started with charts [now](/x/react-charts/getting-started/)!
```

Comment:
```suggestion
[Get started with MUI X Charts now](/x/react-charts/getting-started/)!
```

Taking advantage of the opportunity to also suggest a slightly different copy! 😬